src/X11-distribute-modifications-variant.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="X11-distribute-modifications-variant" name="X11 License Distribution Modification Variant">

```

Comment:
We'll want to add `listVersionAdded="3.16"` since this is a new license. I know it isn't in the original X11 XML since that was a much older one, but for new ID's we're trying to include that property.